bertopic/_bertopic.py

```diff
@@ -984,6 +988,40 @@ def save(self,
             else:
                 joblib.dump(self, file)
 
+    def get_coherence_score(self, documents : List[str], coherence=None):
+
+        """ Getting a coherence score for the topics generated by the model
+        
+        Steps : 
+              * Clean documents using inbuilt preprocessing
+              * Initialize inbuilt CountVectorizer
+              * Build a dictionary using tokens from the tokenized clean documents
+              * Get all topic words from all topics, build a coherence model
+
+
+        """
+        
+        if coherence == None:
+            coherence = 'c_v'
```

Comment:
def get_coherence_score(self, documents : List[str], coherence='c_v') -> float:
setting the defaul coherence to 'c_v'

Reply:
CoherenceModel uses gensim. Should I include gensim in the requirements?